doc/rbd/libvirt.rst

```diff
@@ -237,15 +237,15 @@ commands, refer to `Virsh Command Reference`_.
 #. Define the secret. ::
 
 	sudo virsh secret-define --file secret.xml
-	<uuid of secret is output here>
+	9ec59067-fdbc-a6c0-03ff-df165c0587b8
```

Comment:
Nit: this seems like a step backwards. The original text implies that the output is unknown and that you need to copy/paste it into the next CLI operation on line 248

Reply:
@dillaman I want to align the context in this document. If it's not proper to use show the uuid directly, then it needs to use "{uuid of secret}" in the whole context.

Reply:
I'd vote for `{uuid of secret}` throughout since it's essentially a variable and you should expect to be able to copy-and-paste the commands w/o modification

Reply:
Use "{uuid of secret}" in the whole context now.